TEST

```diff
@@ -34,7 +34,7 @@
 #TEST_ZFSTESTS_DISKSIZE="8G"
 #TEST_ZFSTESTS_ITERS="1"
 #TEST_ZFSTESTS_OPTIONS="-vx"
-#TEST_ZFSTESTS_RUNFILE="linux.run"
+TEST_ZFSTESTS_RUNFILE="bclone.run"
 #TEST_ZFSTESTS_TAGS="functional"
```

Comment:
The CI invoked the test suite like so.  It looks like nothing ran because there is no "functional" tag for the bclone group in the new `bclone.run` file.  I'd suggest adding the tag and updating the PR to give it another try.

```
zfs-tests.sh -vxR -r bclone.run -d /mnt/ -I 1 -T functional
```